torch/lib/ATen/Declarations.cwrap

```diff
@@ -1296,6 +1296,7 @@
   python_name: get_device
   backends:
     - CUDA
+    - SparseCUDA
```

Comment:
This has already been merged but I'm not sure this is the best way to do this. Historically we have used the `aten_sparse` flag on a declaration to enable sparse backend. cc @zdevito

Reply:
I tried that but then it also tries to finde `getDevice` on cpu tensors.

Reply:
https://github.com/zdevito/ATen/blob/master/src/ATen/preprocess_declarations.py#L34-L35

Reply:
@SsnL yes, it may require changing that code a little bit so that if we've only specified the `CUDA` backend, we only enable the `CUDA` and `SparseCUDA` backends.